include/controller/c/webots/robot.h

```diff
@@ -66,15 +66,15 @@ int wb_robot_init();
  */
 #else
 int wb_robot_init_msvc();  // internally, this function just calls wb_robot_init()
-#define wb_robot_init() (setvbuf(stdout, NULL, _IONBF, 0), setvbuf(stderr, NULL, _IONBF, 0), wb_robot_init_msvc())
+#define wb_robot_init() do{setvbuf(stdout, NULL, _IONBF, 0); setvbuf(stderr, NULL, _IONBF, 0); wb_robot_init_msvc()}while(0)
```

Comment:
Shouldn't we add a semi-colon here?
```suggestion
#define wb_robot_init() do{setvbuf(stdout, NULL, _IONBF, 0); setvbuf(stderr, NULL, _IONBF, 0); wb_robot_init_msvc();}while(0)
```

Reply:
yes indeed ! crazy that it build without it, thought